packages/@aws-cdk/aws-backup/test/plan.test.ts

```diff
@@ -70,6 +70,36 @@ test('create a plan and add rules', () => {
   });
 });
 
+test('create a plan and add rules - add BackupPlan.AdvancedBackupSettings.BackupOptions', () => {
+  // GIVEN
+  const vault = new BackupVault(stack, 'Vault');
+  const otherVault = new BackupVault(stack, 'OtherVault');
+
+  // WHEN
+  const plan = new BackupPlan(stack, 'Plan', {
+    windowsVss: true,
+    backupVault: vault,
+    backupPlanRules: [
+      new BackupPlanRule({
+        completionWindow: Duration.hours(2),
+        startWindow: Duration.hours(1),
+        scheduleExpression: events.Schedule.cron({
+          day: '15',
+          hour: '3',
+          minute: '30',
+        }),
+        moveToColdStorageAfter: Duration.days(30),
+      }),
+    ],
+  });
+  // THEN
+  expect(stack).toHaveResource('AWS::Backup::BackupPlan', {
```

Comment:
We have changed the way we unit test since the time you started this PR. This is why your build is failing. Take a look at the code around you, the new line you want is `Template.fromStack(stack).hasResourceProperties('AWS::Backup::BackupPlan', {`